### PR TITLE
dojson: type error unhandled in _get_float

### PR DESCRIPTION
**Pull request by mihaibivol**
Signed-off-by: Mihai Bivol <mihai.bivol@cern.ch>

### CODE REVIEW

File: inspirehep/dojson/institutions/fields/bd1xx.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def location(self, key, value):
     def _get_float(value, c):
         try:
             return float(value[c])
-        except (KeyError, ValueError):
-            return ''
+        except (KeyError, ValueError, TypeError):
```
Comment (jacquerie) [inline inspirehep/dojson/institutions/fields/bd1xx.py]: Whoops. My fault.

**Comment by jacquerie** [comment]
Uh, the idea here was to wait for https://github.com/inspirehep/inspire/issues/170 to be solved on legacy, rather than ignoring the error here. I believe @tomaszgy submitted a similar PR.

**Comment by tomaszgy** [comment]
https://github.com/inspirehep/inspire-next/pull/1272, same thing here :smile:

**Comment by mihaibivol** [comment]
oook. @tomaszgy also has tests. Closing this :)

**Pull request closed without merging**